kettle/update.py

```diff
@@ -22,6 +22,9 @@
 THREADS = 32
 MAX_BAD_RECORDS = 1000
 DAYS_OLD = 1.9
+DAY = 1
+WEEK = 2
```

Comment:
7 ?

Reply:
no idea how I did that lol must have been late